Resources/Private/TypoScript/NodeTypes/NewsFilter.ts2

```diff
@@ -38,4 +38,13 @@ prototype(JohannesSteu.Neos.News:NewsFilter) < prototype(TYPO3.Neos:Content) {
 
         dateFilter = ${dateFilter}
     }
+
+    categoryFilter = ${q(site).find('[instanceof JohannesSteu.Neos.News:Category]').get()}
```

Comment:
Should be named `categories` as it contains a set of catgeories, not a filter itself